mycore-solr/src/main/java/org/mycore/solr/MCRSolrCoreManager.java

```diff
@@ -41,11 +41,11 @@ public final class MCRSolrCoreManager {
 
     private static final Logger LOGGER = LogManager.getLogger(MCRSolrCoreManager.class);
 
-    private static Map<String, MCRSolrCore> CORE_MAP;
+    private static Map<String, MCRSolrCore> coreMap;
```

Comment:
should be final resp. upper case

Reply:
Can't be final because it's set in a try/catch block.